serverside_challenge_2/challenge/spec/lib/utils/energy_price_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe Utils::EnergyPrice do
+  let(:file_path) { Rails.root.join('spec/files/energy_prices_sample.csv') }
+  let(:csv_data) { CSV.read(file_path, headers: true) }
+
+  context 'each_prices' do
+    it '各プランの料金' do
+      prices = described_class.each_prices(csv_data, 20, 150)
```

Comment:
```suggestion
      let(:ampere) { 20 }
      let(:consumption) { 150 }
      prices = described_class.each_prices(csv_data, ampere, consumption)
```

としておけば18行目で150とベタがきしないで済みますね。